src/tsickle.ts

```diff
@@ -740,11 +740,12 @@ class Annotator extends ClosureRewriter {
     if (!classDecl.name) return;
     let className = getIdentifierText(classDecl.name);
 
-    this.emit('\n\n// tsickle -> Closure type declarations\n');
```

Comment:
Don't think it adds something, we have the method name that says the same thing.